nav2_controller/src/controller_server.cpp

```diff
@@ -62,8 +62,8 @@ ControllerServer::ControllerServer(const rclcpp::NodeOptions & options)
 
   // The costmap node is used in the implementation of the controller
   costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>(
-    "local_costmap", std::string{get_namespace()}, "local_costmap");
-
+    "local_costmap", std::string{get_namespace()}, "local_costmap", get_parameter(
```

Comment:
Last nit-pick and I'll merge this: Put the `get_parameter(` on the new line, its weird to have the line extension break there. I suspect that was a linter auto-fix. You can place `get_parameter("use_sim_time").as_bool());` all on the second line. As well for planner server, this is just a readability thing.

Reply:
Done. Yeah it was just automatic uncrustify. I'm pretty sure this can be avoided with some custom uncrustify config